airlock/business_logic.py

```diff
@@ -513,9 +513,11 @@ class RequestFile:
     job_id: str
     commit: str
     repo: str
+    release_output_checker: str
```

Comment:
I guess this is non-nullable because we just use an empty string instead of NULL? Feels like it might be conceptually neater and more consistent if it was nullable though.

Reply:
yes, I went with empty string, could be NULL though? probably conceptually makes more sense - I'll change.